azure-devops/publish-tests.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation.

```

Comment:
I wouldn't rename this file to `publish-tests`. For the native build modes this file just runs the tests.